.circleci/config.yml

```diff
@@ -13,6 +13,16 @@ jobs:
       - image: circleci/php:7.2-node-browsers
     steps:
       - checkout_with_workspace
+      - run:
+          name: Npm 7
+          command: |
+            set +e
+            set +x
+            export NVM_DIR="$HOME/.nvm" && (
+              git clone https://github.com/nvm-sh/nvm.git "$NVM_DIR"
+              cd "$NVM_DIR"
+            ) && \. "$NVM_DIR/nvm.sh" --no-use
+            nvm install --latest-npm
```

Comment:
we can probably ditch this, as it didn't solve the problem in CI

Reply:
Yup, removed